packages/icons/src/library/metabox.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/primitives';
+
+const metabox = (
+	<SVG width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg">
+		<Path
+			fill-rule="evenodd"
```

Comment:
When I apply the PR I see a JavaScript warning in the block editor:
`Warning: Invalid DOM property `fill-rule`. Did you mean `fillRule`?`

Reply:
Ah, thanks you :)